sci-physics/lhapdf/lhapdf-6.5.5-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DOCS_BUILDER="doxygen"
+DOCS_DEPEND="
+	dev-texlive/texlive-bibtexextra
+	dev-texlive/texlive-fontsextra
+	dev-texlive/texlive-fontutils
+	dev-texlive/texlive-latex
+	dev-texlive/texlive-latexextra
+"
+inherit python-single-r1 docs
+
+MY_PV=$(ver_cut 1-3)
+MY_PF=LHAPDF-${MY_PV}
+
+DESCRIPTION="Les Houches Parton Density Function unified library"
+HOMEPAGE="https://lhapdf.hepforge.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/hepcedar/lhapdf"
+else
+	SRC_URI="https://www.hepforge.org/downloads/lhapdf/${MY_PF}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${MY_PF}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="examples +python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	dev-cpp/yaml-cpp
+	python? ( ${PYTHON_DEPS} )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	python? (
+		$(python_gen_cond_dep '
+			>=dev-python/cython-0.19[${PYTHON_USEDEP}]
+		')
+	)
+"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_configure() {
+	local -x CONFIG_SHELL="${EPREFIX}/bin/bash"
+	econf \
+		--disable-static \
+		--with-yaml-cpp="${EPREFIX}/usr" \
+		$(use_enable python) \
+		$(use_enable doc doxygen)
+}
```

Comment:
You could use here the `emyeconfargs=()` method instead.

Reply:
Sure, I don't see too much of a difference in the end. What is the advantage of `myeconfargs`? I save the `\` per line, but I ocassionally have to double quote like here  https://github.com/gentoo/sci/pull/1340/files#diff-5916d6d64a6217ceba12574384d193eb0558c33cb94e2afdbee4262d9b4217ccR37. Missing the double quote seems more error prone than missing the `\`.

Reply:
Looking at above linked ebuild https://github.com/gentoo/sci/blob/41156791229979561f337467458acfa137338241/sci-physics/samurai/samurai-2.1.1.ebuild#L33 is probably wrong when `$ESYSROOT` has a space.

Reply:
> Sure, I don't see too much of a difference in the end. What is the advantage of myeconfargs?

Nothing, it's just a matter of aesthetics, it looks more similar to how things are done with meson or cmake.

Reply:
Actually, now that you mention it. Shouldn't the `EPREFIX`s here be `BROOT` and `ESYSPREFIX` instead?

Reply:
Based on the number of `EPREFIX` (17) for `CONFIG_SHELL` vs `BROOT` (86) in ::gentoo). There is no `ESYSPREFIX` listed here https://devmanual.gentoo.org/ebuild-writing/variables/. Also `BROOT` is given as (EAPI=7).

Reply:
Sorry, I meant ESYSROOT, the page you linked says that is where the DEPENDs are. Same for bash, it is a BDEPEND, and the devmanual says those are on the BROOT.

Reply:
I see both being used in ::gentoo for headers and libs, but `ESYSROOT` seems more correct indeed.